src/agent/onefuzz-task/src/tasks/fuzz/libfuzzer/common.rs

```diff
@@ -236,12 +236,7 @@ where
                 .for_each(|d| inputs.push(&d.local_path));
         }
 
-        let fuzzer = LibFuzzer::new(
-            &self.config.target_exe,
-            self.config.target_options.clone(),
-            self.config.target_env.clone(),
-            &self.config.common.setup_dir,
-        );
+        let fuzzer = L::from_config(&self.config).await?;
```

Comment:
Opportunistic fix. We should only ever be creating `LibFuzzer` values via `L::from_config()`.